docs/blog/newsletter-december-2019.rst

```diff
@@ -0,0 +1,127 @@
+.. post:: December 03, 2019
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2019
+#########################################
+
+Hi everybody! It's Beth here, with the last issue of 2019. And what a year it's been! Four conferences (including our first time in `Vilnius </conf/vilnius/2019/>`__, and the recently-wrapped-up `Australia conference </conf/australia/2019/>`__), more meetups than you can shake a stick at, and an ever-busier community on Slack.
+
+And there's plenty more to look forward to in 2020. If you like taking the time  to think about the big picture over the holidays, we'd love the gift of your talk proposal for Portland 2020 - the `CFP </conf/portland/2020/cfp/>`__ is open until 10th January. We’re also hoping to have another US conference next year, closer to folks on the East coast - but this is still unconfirmed. We'll keep you posted.
+
+So that's all on the news front. We’ve had another month of great conversations and insights from the Write the Docs Slack - enjoy!
+
+----------------------------------------------------
+Documentation feedback widgets - thumbs up or down?
+----------------------------------------------------
+
+Getting user feedback on your documentation can be challenging (and not just emotionally!). This month, our community discussed options for asking for that feedback on our help sites.
```

Comment:
Maybe "...challenging, and not just emotionally!" to avoid the punctuation-close paren-punctuation thing.

Reply:
agreed, fixed